lib/inspec/resources/service.rb

```diff
@@ -304,10 +304,12 @@ def info(service_name)
 
       # LoadState values eg. loaded, not-found
       installed = params["LoadState"] == "loaded"
+      startname = params["User"] != "" ? params["User"] : nil
```

Comment:
Could we do this instead:

`startname = params["User"]`

I think it would be nicer to have a empty string since we're expecting a string if its set.

Reply:
Agreed. This complexity isn't necessary.

Reply:
Updated